tests/cpp/operator/test_dequantize_mxfp8.cu

```diff
@@ -61,18 +61,41 @@ void compute_ref_x1(const InputType* input,
                     const size_t block_size_X,
                     const size_t scales_stride)
 {
-    const size_t blocks_Y = (rows + block_size_Y - 1) / block_size_Y;
-    const size_t blocks_X = (cols + block_size_X - 1) / block_size_X;
-
-    for (size_t ii = 0; ii < blocks_Y; ++ii) {
-        const size_t i_min = ii * block_size_Y;
-        const size_t i_max = std::min((ii + 1) * block_size_Y, rows);
-        for (size_t jj = 0; jj < blocks_X; ++jj) {
-            const size_t j_min = jj * block_size_X;
-            const size_t j_max = std::min((jj + 1) * block_size_X, cols);
-            const size_t scale_idx = ii * scales_stride + jj;
-            dequantize_block<InputType, OutputType>(
-                input, output, scales, scale_idx, i_min, i_max, j_min, j_max, cols);
+    const size_t tile_size_Y = std::max(32lu, block_size_Y);
+    const size_t tile_size_X = std::max(64lu, block_size_X);
+    const size_t tiles_num_Y = (rows + tile_size_Y - 1) / tile_size_Y;
+    const size_t tiles_num_X = (cols + tile_size_X - 1) / tile_size_X;
+    const size_t blocks_per_tile_Y = tile_size_Y / block_size_Y;
+    const size_t blocks_per_tile_X = tile_size_X / block_size_X;
+
+    #pragma omp parallel proc_bind(spread)
+    {
+        #pragma omp for schedule(static)
```

Comment:
Could we consolidate this into a single `omp parallel for`?

Reply:
Done in [5d0a899](https://github.com/NVIDIA/TransformerEngine/pull/1534/commits/5d0a899460a299afb1504cafa422f88f16d7197d)